### PR TITLE
Add testing

### PR DESCRIPTION
**Pull request by IanLee1521**
This adds testing options, locally with tox and via Travis. The travis testing will require @h-m-f-t to enable that integration.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@ docopt
 publicsuffix
 py3dns
 pyspf==2.0.11
+tox
```
Comment (IanLee1521) [inline requirements.txt]: Good catch, thanks @jsf9k !

**Comment by jsf9k** [comment]
I'll leave it to @h-m-f-t to do the final approval, since he has the ability to turn on Travis.

**Comment by IanLee1521** [comment]
No worries. And thanks for fixing the merge conflict, I was trying to do that same thing here, but you were faster than me by 1 minute... ;)

**Comment by jsf9k** [comment]
I'm going to go ahead and merge this, so the pull request stops requiring merges with `origin/master`.  We still need @h-m-f-t to turn on Travis-CI for the repository.  I don't have sufficient permissions to do it.

**Pull request merged**